### PR TITLE
check permissions when creating a package

### PR DESCRIPTION
**Pull request by btel**
related to #160

### CODE REVIEW

File: quetz/tests/test_main.py [hunk 1]
```diff
@@ -393,3 +393,17 @@ def test_channel_action_reindex(auth_client, public_channel, expected_code):
...
+
+@pytest.mark.parametrize(
+    "channel_role,expected_code",
+    [("owner", 201), ("maintainer", 201), ("member", 403), (None, 403)],
+)
+def test_add_package_permissions(auth_client, public_channel, expected_code):
+
+    response = auth_client.post(
+        f"/api/channels/{public_channel.name}/packages",
+        json={"name": "test-package", "summry": "none", "description": "none"},
```
Comment (wolfv) [inline quetz/tests/test_main.py:406]: typo in summry?

**Comment by wolfv** [comment]
lgtm!

File: quetz/tests/test_main.py (hunk 1 above)
Comment (btel) [reply quetz/tests/test_main.py:406]: true, but still it passes :-)

```suggestion
        json={"name": "test-package", "summary": "none", "description": "none"},
```

**Comment by btel** [comment]
good to merge @wolfv

**Comment by btel** [comment]
this was fast, do you have a bot to merge PRs? :smile: @wolfv

**Comment by wolfv** [comment]
not yet, not yet :)

**Pull request merged**